README.md

```diff
@@ -170,6 +171,17 @@ n.reorder_edges(e_out) # Permute final tensor as necessary
 print(tf.norm(tf.matmul(a,b) - n.get_tensor()))
 ```
 
+## Different backend support.
+Currently, we support Tensorflow, JAX, and NumPy for the TensorNetwork backends. 
```

Comment:
Tensorflow --> TensorFlow

Suggested wording change: "..for the TensorNetwork backends." --> "...as TensorNetwork backends."